sdk/src/main/java/com/bugsnag/android/Bugsnag.java

```diff
@@ -19,7 +19,8 @@
 @SuppressWarnings("checkstyle:JavadocTagContinuationIndentation")
 public final class Bugsnag {
 
-    @Nullable
```

Comment:
why did the client lose its `@nullable` annotation?

Reply:
The main reason we add nullability annotations is to provide information on our public APIs - as this field isn't public and will nearly always be non-null, I felt this annotation is somewhat misleading.